src/components/popover/popover.component.scss

```diff
@@ -1,11 +1,66 @@
-:host {
+.popover {
     display: flex;
     flex-direction: column;
-    max-width: none;
     padding: 0;
+    max-width: 568px;
```

Comment:
The agent will be localized in the future, so it is possible that there is a language which uses longer words for tabs, so the popover should grow, but can't because of this magic limit.